src/Config/SessionConfig.php

```diff
@@ -133,7 +133,18 @@ public function setStorageOption($storageName, $storageValue)
                 break;
         }
 
+        $sessionAlreadyStarted = false;
+        if (session_status() == PHP_SESSION_ACTIVE) {
+            $sessionAlreadyStarted = true;
+            session_write_close();
```

Comment:
Honestly, this seems really problematic, considering your original use case as demonstrated in #104. It means that you are _starting_, _stopping_, and _restarting_ your session _on every request_, resulting in I/O _even if no data is ever written or accessed_. This may not seem like a big deal, but it can lead to longer response times (due to I/O of writing the session), and potential concurrency issues.

Wouldn't it be more performant and less prone to error to do as I suggested in that issue and inject the fully configured session manager into your container during instantiation? Have you tried the solution I posted on that issue?

Reply:
Yes, I tried your solution, but that still got issue, for example, on call rememberMe() or clearIdentity() after login success that requires changes cookie lifetime, while session already active.

Reply:
also when calling `sessionmanager->forgetMe()` as it set cookie_lifetime to 0, while session already started